jiradata/TransitionsFuncs.go

```diff
@@ -9,10 +9,17 @@ import (
 // or nil
 func (t Transitions) Find(name string) *Transition {
 	name = strings.ToLower(name)
+	matches := []Transitions{}
 	for _, trans := range t {
-		if strings.Contains(strings.ToLower(trans.Name), name) {
+		if strings.Compare(strings.ToLower(trans.Name), name) == 0 {
```

Comment:
this is more commonly written as `if strings.ToLower(trans.Name) == name {`

Reply:
Done!